course_flow/models.py

```diff
@@ -221,6 +241,10 @@ class Activity(models.Model):
         Strategy, through="StrategyActivity", blank=True
     )
 
+    columns = models.ManyToManyField(
+        Column, through="ColumnActivity", blank=True
```

Comment:
I think this should be a foreign key to activity.  If I understand correctly, I don't think a certain column could ever be used by two different activities.

Reply:
Alternatively, columns could be referenced on the activity model as manytomany. I tried to keep with the style of children always being referenced on parents. e.g. Activity references strategy, because strategy "within" activity.

Reply:
I'm not sure I understand your point here. I made sure to follow the logic you used for the strategies/weeks. You're right about a column not being used by two different activities, but that's no different than a week or a strategy.

For your second comment, could you explain in more detail how that is different than what is currently there?

Reply:
For the first part, I think what @aahilgert is proposing is to reverse the relationship and add a fk on Column pointing at Activity instead of a m2m on Activity pointing at a list of Columns.

Reply:
If so, shouldn't this be done for weeks/"strategies" as well?